### PR TITLE
New JSON files for schema and examples

### PR DESCRIPTION
**Pull request by sei-vsarvepalli**
Request review and merge of this branch for updates to the /data folder - adding JSON schemas and examples of JSON files under folders "schema" and "computed".  These have been tested with JSON Schema tool for version 4.0 JSON schema. 

Note to remember and fix later: 
Even the latest JSON Schema (7) does not have a good way to represent Epoch in milliseconds and relating it to a timestamp.  So for now, Epoch timestamp remains an integer and the date field remains a normal date-time field - potentially using ISO 8601 time format.

### CODE REVIEW

**Review by j--- — APPROVED** [review]

**Comment by j---** [comment]
I think this closes the substantive part of #65. However, to actually fully handle #65, we'll also need to edit the .md to talk about the JSON schema (which I plan to handle).

**Comment by sei-vsarvepalli** [comment]
> I think this closes the substantive part of #65. However, to actually fully handle #65, we'll also need to edit the .md to talk about the JSON schema (which I plan to handle).

Hello Jono,

I have a simple README.md, it will be good to make it more exhaustive and clear about how it will be used.  The schema will probably also be iterative and may require improvements.  I will wait for @ahouseholder thoughts also.

Thanks
Vijay

File: data/schema/SSVC_Computed_v2.schema.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	    "description": "Computed score short representation such as SSVCv2/Ps:Nm/T:T/U:E/1605040000/ for a vulnerability with no or minor Public Safety Impact,  total Technical Impact, and efficient Utility, which was evaluated on Nov 10, 2020.",
+	    "type": "string"
+	},
+	"timestamp" : {
+	    "description": "Date and time in ISO format  ISO 8601 format",
+	    "type": "string",
+	    "format": "date-time"
+	},
+	"timestamp_epoch_ms" : {
+	    "description": "Date and time in epocj utc millisec",
```
Comment (ahouseholder) [inline data/schema/SSVC_Computed_v2.schema.json:29]: "epocj" should be "epoch"

Comment (sei-vsarvepalli) [reply data/schema/SSVC_Computed_v2.schema.json:29]: Thanks for this catch.  There is a little more consolidation of information I noticed in the Provision JSON schema.  I will update all this and push a new PR soon out.

**Review by ahouseholder — CHANGES_REQUESTED** [review]
- [ ] _timestamp_ should be ISO 8601 with time zone to make it easier for receiver parsing
- [ ] _timestamp_epoch_ms_ appears redundant to _timestamp_ and is easy enough to get on the receiver end once _timestamp_ has been parsed. I'd prefer just having the one _timestamp_ field. 
- [ ] If characters in ISO8601 are a problem for the shorthand notation (which appears to append a timestamp to an ssvc string), I'd just make it positional: e.g., YYYYmmddHHMMSS. I don't see a need for subsecond resolution, and human-readability has value.

**Comment by sei-vsarvepalli** [comment]
> * [ ]  _timestamp_ should be ISO 8601 with time zone to make it easier for receiver parsing
> * [ ]  _timestamp_epoch_ms_ appears redundant to _timestamp_ and is easy enough to get on the receiver end once _timestamp_ has been parsed. I'd prefer just having the one _timestamp_ field.
> * [ ]  If characters in ISO8601 are a problem for the shorthand notation (which appears to append a timestamp to an ssvc string), I'd just make it positional: e.g., YYYYmmddHHMMSS. I don't see a need for subsecond resolution, and human-readability has value.

Allen,
Thanks very much for the review and comments! 

One question I have is YYYYmmddHHMMSS should that be in UTC? 

I have a few more changes that I will consolidate into this PR (include your changes as well) this afternoon. Appending the timestamp epoch for short form of ssVC score  is mentioned in the below link. @j---  may need to update that document as well for ease of reading date-time in the shorthand score representation. 

https://github.com/CERTCC/SSVC/blob/main/doc/version_1/047_treesForVulMgmt_4.md

**Comment by sei-vsarvepalli** [comment]
I am going to completely close this one and start a new PR, this branch is now all confusingly messed up.

**Pull request closed without merging**